### ISSUE TITLE
Clarify significance of grey nodes in documentation

### ISSUE THREAD

**Issue by cmiya** [cwrc/HuViz#22]
-	What is the significance of the node colour? Most appear to be gray.
-	If the node colours reflect the class category this needs to be clarified (perhaps with a hover help bubble).
-	Kathryn worked with the poetess and bronte datasets and only saw grey → could this be a bug??

**Comment by ilovan** [comment]
I'm thinking it may be the different ways in which browsers handle instructions from the HuViz code?
We also have to be very careful not to duplicate the transmission of the same type of information in two different ways.  (e.g. intensity of colour and size of bubble to express number of occurences)

**Comment by smurp** [comment]
Nodes are colored according to which class they are an immediate instance of.  HuViz can only know what class most thing are by seeing which predicates they are the object of.  The new ontology should do a better job of supporting such class-instance discovery and hence coloring of nodes.

**Comment by cmiya** [comment]
Ah, okay. So it's an issue that should be fixed with the new ontology. Just to be clear - What do you mean by which class they are an "immediate" instance of? Could a node be a member of multiple classes, in which case how would the colour be determined?

**Comment by kathrynholland** [comment]
Colours are fine for me now.

**Comment by ilovan** [comment]
We need to keep in mind accessibility when conveying information though colour. 
Resources: http://www.inpixelitrust.fr/blog/en/tips-create-accessible-color-palette/

**Comment by smurp** [comment]
@cmiya I am using *immediate class instance" as a synonym for "direct class instance* which means the  most specific class that the entity is an instance of.  There is the assumption in such terminology that the entity is a direct instance of only one class and that the distinction being made is between that most immediate class and other classes up the inheritance tree from that class.  Indeed, there is a possible source of error in the system at the moment, if nodes are the object of predicates in separate branches of the taxonomy, ie if there is *multiple inheritance* going on.  I was aware of this possibility of course but don't recall how I handled such cases.  I suspect that I ignored them on the assumption that the taxonomy wasn't bushy enough to be exposing us to such risks much.

Proposal:
* If a node is a direct instance of multiple classes on different branchs of the taxonomy then the node should be colored with pie pieces of the appropriate colors
